### ISSUE TITLE
[Web] Use video formats for animated content

### ISSUE THREAD

**Issue by dubrie** [Green-Software-Foundation/green-software-patterns#62]
**Describe the pattern you'd like to propose**
Large GIFs are inefficient for delivering animated content. By converting large GIFs to videos, you can save big on users' bandwidth. Consider using MPEG4/WebM videos for animations and PNG/WebP for static images instead of GIF to save network bytes.

**Describe specific emission impact from this pattern**
Reduced energy consumption by reducing download size and increased page performance

**References to this pattern**
https://web.dev/efficient-animated-content/

**Additional context**
Add any other context or screenshots about the proposed pattern here.

**Comment by markus-ntt-seidl** [comment]
I'm taking over this.

**Comment by markus-ntt-seidl** [comment]
I would rephrase this, as the title is not true IMHO.
True, you could use Video for animations, but there are CSS (+JS), SVG and stuff to also animate content. Vector animation is sharper for less bandwidth and maybe more efficient, depending on the content.
So I'm going to rephrase this as "Deprecate GIFs for animated content".

Funny side note: "Back in the days" 😄 , between gifs and videos, when video was still too expensive, there was something like packing changes on images into an image to display videos. I just saw that the frontpage of [Sublime Text](https://www.sublimetext.com/) is still using it as the hero animation on the page ( one "frame" looks like [this](https://www.sublimetext.com/screencasts/osx_image_2_dark@2x_packed.png) ). Highly optimized and, maybe ❓  , better than video.
